.gitlab/benchmarks/macrobenchmarks.yml

```diff
@@ -394,6 +395,177 @@ profiler_cpu_timer_create-arm64:
   variables:
     NATIVE_PROFILER_PATH: "dd-trace-dotnet/tracer/tracer-home-linux-arm64/linux-arm64/Datadog.Trace.ClrProfiler.Native.so"
     TRACER_HOME_PATH: "dd-trace-dotnet/tracer/tracer-home-linux-arm64"
+    COR_ENABLE_PROFILING: 1
+    CORECLR_ENABLE_PROFILING: 1
+    DD_PROFILING_ENABLED: 1
+    DD_PROFILING_WALLTIME_ENABLED: 0
+    DD_PROFILING_CPU_ENABLED: 1
+    COMPlus_EnableDiagnostics: 1
+    ENDPOINT: "hello"
+
+.benchmarks-win:
+  stage: benchmarks-win
+  needs: ["check_azure_pipeline"]
+  tags: ["runner:docker"]
```

Comment:
I think runner: docker is deprecated nowadays and we should use arch:amd64, unless there is really a compelling reason to keep it.